modules/service/src/main/scala/lucuma/odb/service/TelluricTargetsDaemon.scala

```diff
@@ -74,6 +76,12 @@ object TelluricTargetsDaemon:
           val (pending, meta) = result
           info"Resolved telluric for ${pending.observationId}: $meta"
         .void
+        .attempts(Stream.constant(WaitToRestart))
+        .evalTap:
+          case Left(e)  => error"Telluric daemon error: ${e.getMessage}, restarting in $WaitToRestart..."
```

Comment:
I don't think I really understand this. It seems like `attempts` just waits for WaitToRestart and then tries to pull from the stream again. Which isn't quite the same as restarting the daemon, is it? Or is it equivalent in this case?